### ISSUE TITLE
Redis error: ERR Client sent AUTH, but no password is set

### ISSUE THREAD

**Issue by jinnysunny** [openware/peatio#133]
When trying to access the markets:

```
Started GET "/api/v2/k_with_pending_trades.json?market=btccny&limit=768&period=60&trade_id=0" for ::1 at 2017-12-19 19:54:02 +0200

Redis::CommandError - ERR Client sent AUTH, but no password is set:
  app/api/api_v2/helpers.rb:95:in `get_k_json'
  app/api/api_v2/k.rb:25:in `block in <class:K>'
  lib/middleware/security.rb:11:in `call'
  lib/middleware/i18n_js.rb:9:in `call'
```

**Comment by jinnysunny** [comment]
Needed to strip the password requirement in `kline_db.rb` to make it work. Probably my own redis installation needs to be reinstalled.

**Comment by yivo** [comment]
@propercoil I have this error too but I have caught it in `k.rb`:

```
/Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:119:in `call': ERR Client sent AUTH, but no password is set (Redis::CommandError)
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:100:in `block in connect'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:291:in `with_reconnect'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:98:in `connect'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:363:in `ensure_connected'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:219:in `block in process'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:304:in `logging'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:218:in `process'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis/client.rb:118:in `call'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis.rb:1241:in `block in lindex'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis.rb:45:in `block in synchronize'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/2.4.0/monitor.rb:214:in `mon_synchronize'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis.rb:45:in `synchronize'
	from /Users/yaroslav/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/redis-4.0.1/lib/redis.rb:1240:in `lindex'
	from lib/daemons/k.rb:18:in `last_ts'
	from lib/daemons/k.rb:23:in `next_ts'
	from lib/daemons/k.rb:114:in `block in <main>'
	from lib/daemons/k.rb:113:in `each'
	from lib/daemons/k.rb:113:in `<main>'
```

**Comment by yivo** [comment]
BTW I have clean redis installation with default configs

**Comment by yivo** [comment]
@propercoil The problem is that default application.yml is shipped with `REDIS_PASSWORD: changeme` which is sent when redis client connect to the server. But if server doesn't expect passsword to be sent when connecting it will respond with an error like your's. So the fix is to set `REDIS_PASSWORD` to nil. 

PR is ready which fixes it #143
